demo/smith_fuzz/ci/mzbuild.yml

```diff
@@ -0,0 +1,13 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+name: smith-fuzz
```

Comment:
I'm not familiar enough with our current ci infra so not sure what exactly gets built as a result of these files -- make sure its what I want